tasks/install-manageiq-conversion-host.yml

```diff
@@ -6,7 +6,7 @@
     user: "{{ manageiq_username }}"
     password: "{{ manageiq_password }}"
     method: "POST"
-    validate_certs: true
+    validate_certs: false
```

Comment:
I'd rather see this as configuration option that defaults to "true".

Reply:
Good point. It should be better now.